src/renderer.zig

```diff
@@ -592,9 +598,9 @@ pub const MenuBackGround = struct {
 		lastTime = newTime;
 		const viewMatrix = Mat4f.rotationZ(angle);
 		pipeline.bind(null);
-		updateViewport(main.Window.width, main.Window.height, 70.0);
-		defer updateViewport(Window.width, Window.height, settings.fov);
-
+		if(lastFov != 70.0) {
```

Comment:
I think it would make sense to put the `lastFov != xxx` check into the function.

Reply:
The whole takeBackgroundImage should only be run when in the game, so it might make sense to change functions to fit that.

Reply:
I've changed the function.

I added a `std.debug.assert` to `renderer.render` too. If I misunderstand how to use that, let me know. But my intention was to prevent people from running that function if game.world is null (and the assert should prevent us from making changes that don't obey this)

Reply:
Yes, assert is good there.